contracts/Scroll_SpokePool.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import "./SpokePool.sol";
+import "@scroll-tech/contracts/L2/gateways/IL2GatewayRouter.sol";
+import "@scroll-tech/contracts/libraries/IScrollMessenger.sol";
+
+/**
+ * @title Scroll_SpokePool
+ * @notice Modified SpokePool contract deployed on Scroll to facilitate token transfers
+ * from Scroll to the HubPool
+ */
+contract Scroll_SpokePool is SpokePool {
+    IL2GatewayRouter public l2GatewayRouter;
```

Comment:
Should add comments to any state variables

Reply:
Done: https://github.com/across-protocol/contracts-v2/pull/381/commits/a44f7569b26cccaba4418aad00cb7d487477a50b